bin/install.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+appendIfNotExist() {
+  if ! grep -Fxq "$1" $HOME/.zshrc; then
```

Comment:
`if [[ ! $() ]]` to be consistent

Reply:
This if statement is testing the return code of the command, `if [[ ! $() ]]` captures the stdout.

Reply:
True, my bad.